index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+    <main>
+        <section class="container">
+                <nav class="nav_bar">
+                        <span class="myLogo"><a href="#"><img src="image/My_Logo.svg" alt="My Logo"></a></span>
+                        <span class="contact"><a href="#"><img src="image/Icon - Menu.svg" alt="Contact Us"></a></span>
+                </nav>
```

Comment:
You have done great by including the `nav` tag. It would be more semantic if the logo were outside the `nav` tag. The navigation is usually used to link to different sections of the website.

Also, we need to include a header tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

So, I would recommend that the `nav` tag changes to `header` tag while the second span changes to nav to hold the navigation menu.